js/script.js

```diff
@@ -0,0 +1,80 @@
+const speakers = [
+  {
+    name: 'Matias Medina',
+    title: 'Singer',
+    description: 'Matias was born in Mendoza, he has a lot of time like singer',
+    image: 'speaker_01.png',
+  },
+  {
+    name: 'Diego Viale',
+    title: 'Violinist',
+    description: 'Diego is one of the founder of the band, he study the instrument with Lele Lovato, He has more than 20 years of experience with the violin',
+    image: 'speaker_02.png',
+  },
+  {
+
+    name: 'Daniel Sage',
+    title: 'Bass',
+    description: 'Daniel is from USA, but the music send to Argentina',
+    image: 'speaker_03.png',
+  },
+  {
+    name: 'Franco Aguero',
+    title: 'Percussion',
+    description: 'Franco is an excellent dancer, director for the Argentinian Malambo Group.',
+    image: 'speaker_04.png',
+  },
+  {
+    name: 'Maximiliano Guerrero',
+    title: 'Baterist',
+    description: 'Maxi is one of the most important baterist from Colombia, he was born in Bogota',
+    image: 'speaker_05.png',
+  },
+  {
+    name: 'Sebastian Maure',
+    title: 'Guitarist',
+    description: 'Sebastian started with another style of music, he play Jazz and Rock song in the beginning',
+    image: 'speaker_06.png',
+  },
+  {
+    name: 'Leonardo Ibacache',
+    title: 'Guitarist',
+    description: 'Leonardo is the youngest guy in the group, This is your first experience in a Tour',
+    image: 'speaker_07.png',
+  },
+  {
+    name: 'Guillermo Gomez',
+    title: 'Manager',
+    description: 'Guillermo is one of the most important manager of Argentina.',
+    image: 'speaker_08.png',
+  },
+];
+const menuButton = document.querySelector('.menu-button');
+const menu = document.querySelector('.menu');
+const closeButton = document.querySelector('.menu-close');
+const menuItems = document.querySelectorAll('.menu-item');
+const speakersContainer = document.querySelector('.speakers-content');
+const toggleMenu = () => {
+  menu.classList.toggle('display-none');
+};
+menuButton.addEventListener('click', toggleMenu);
+closeButton.addEventListener('click', toggleMenu);
+menuItems.forEach((menuItem) => {
+  menuItem.addEventListener('click', toggleMenu);
+});
+speakers.forEach((speaker, index) => {
```

Comment:
When I navigate to the about page, there is an error in the console.  Kindly fix it. ⬇️
![image](https://user-images.githubusercontent.com/33268927/196942957-7e1bf08d-23c1-4682-a2fd-6e98072c66d1.png)
Hint: This error occurs because the speaker's container doesn't exist on the about page and javascript is trying to find the speaker's container. You can fix this error by rendering the speakers conditionally. If speakers container exists, only then render speakers.